jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -139,11 +150,17 @@ void setDisableUpdateCheck(boolean disableUpdateCheck) {
       }
 
       // Check for update
-      HttpURLConnection connection = (HttpURLConnection) new URL(versionUrl).openConnection();
+      FailoverHttpClient httpClient = new FailoverHttpClient(true, false, log);
```

Comment:
Actually, maybe it's better to ignore logs?

`new FailoverHttpClient(true, false, ignored -> {})`

Reply:
True, I'll submit another PR.